pkg/grpc/universalapi/api_workflow.go

```diff
@@ -0,0 +1,201 @@
+/*
+Copyright 2023 The Dapr Authors
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+    http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package universalapi
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+	"time"
+
+	"google.golang.org/grpc/codes"
+	"google.golang.org/grpc/status"
+
+	"github.com/dapr/components-contrib/workflows"
+	"github.com/dapr/dapr/pkg/messages"
+	runtimev1pb "github.com/dapr/dapr/pkg/proto/runtime/v1"
+)
+
+func (a *UniversalAPI) GetWorkflowAlpha1(ctx context.Context, in *runtimev1pb.GetWorkflowRequest) (*runtimev1pb.GetWorkflowResponse, error) {
+	if in.InstanceId == "" {
+		err := status.Errorf(codes.InvalidArgument, messages.ErrMissingOrEmptyInstance)
+		a.Logger.Debug(err)
```

Comment:
Do we need these debug logs? Do we not have common error logging code for universal APIs? There's a lot of redundancy here so I feel like it would make much more sense to have a single place where we do error logging for all APIs.

Reply:
Sadly we cannot have common error logging because in some cases errors are returned by a helper function which could do its own logging.
At least it's only 1 statement :)